0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -115,12 +74,20 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
-  "type": "payee",
-  "account": "ilpdemo.red.bob",
-  "currency_code": "USD",
-  "currency_symbol": "$",
-  "name": "Bob Dylan",
-  "image_url": "https://red.ilpdemo.org/api/receivers/bob/profile_pic.jpg"
+  "destination_account": "example.ilpdemo.red.bob",
+  "shared_secret": "6jR5iNIVRvqeasJeCty6C-YB5X9FhSOUPCL_5nha5Vs",
+  "maximum_destination_amount": "100000",
+  "minimum_destination_amount": "10",
+  "ledger_info": {
+    "currency_code": "USD",
+    "currency_symbol": "$",
```

Comment:
ilp-kit and other node-based UI applications should start using some standard registry like https://www.npmjs.com/package/currency-symbol-map, and people can then send pull requests to that npm module to make their custom currencies have fancy symbols. If there is a need, we can create a cross-language registry somewhere. But IMHO, making the UI appearance part of the protocol breaks separation of concerns.